apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -5911,12 +5915,10 @@ data_code_to_reason([1003])      -> <<"invalid_pubkeys">>;
 data_code_to_reason([1004])      -> <<"call_not_found">>;
 data_code_to_reason([1005])      -> <<"broken_encoding: accounts">>;
 data_code_to_reason([1006])      -> <<"broken_encoding: contracts">>;
+data_code_to_reason([1007])      -> <<"contract_init_failed">>;
```

Comment:
Is this meant to integrate https://github.com/aeternity/protocol/blame/3a00b538ed47efa6ab7bb205f07456e06134ff44/node/api/channel_ws_api_json-rpc.md#L179 ?

Reply:
Addressed in aeternity/protocol#320